frontend/src/components/UpcomingReminderGallery.tsx

```diff
@@ -0,0 +1,60 @@
+import {useEffect, useState} from "react";
+import {Reminder} from "../types/Reminder.ts";
+import axios from "axios";
+
+export default function UpcomingReminderGallery() {
+    const [upcomingReminders, setUpcomingReminders] = useState<Reminder[]>([]);
+    const [showUpcomingReminder, setShowUpcomingReminder] = useState<boolean>(false);
+
+    useEffect(() => {
+
+        getUpcomingReminders();
+
+        const interval = setInterval(() => {
+            getUpcomingReminders();
+        }, 60000);
+
+        return () => clearInterval(interval); //Cleanup the interval on unmount
+
+    }, []);
+
+
+    const getUpcomingReminders = () => {
+        axios.get("/api/reminders/upcoming")
+            .then(response => {
+                if(response.data.length > 0) {
+                    //setUpcomingReminders(response.data);
+                    setUpcomingReminders(preReminders => [...preReminders, ...response.data]);
+                    setShowUpcomingReminder(true);
+                }
+            })
+            .catch(error => console.error("Error getting upcoming reminders", error))
+            .finally(() => {console.log("getUpcomingReminder_successful")})
```

Comment:
You should remove this line. If you want to you can implement frontend logging with info or debug level.